recipes-soletta/soletta/soletta_git.bb

```diff
@@ -105,6 +105,9 @@ do_configure_append() {
        sed -i -e 's/^ *FLOW_NODE_TYPE_GTK=.*/# FLOW_NODE_TYPE_GTK is not set/g' ${S}/.config
    fi
 
+   sed -i "s/^\(CONFIG_LDFLAGS=\).*/\1\"${TARGET_LDFLAGS}\"/g" ${S}/.config
```

Comment:
@barbieri git blame does not say why 'unset' is there. That line was there in oe-core initial commit.

Perhaps the thinking is kconfig is expected to deal with its own flags. Luckily, we have ```CONFIG_LDFLAGS``` in soletta to use. Any other variable would require soletta patching in order to make soletta to use those. I was also thinking if it would make sense to change dependency-resolver.py/dependencies.json to use hash-style but that does not solve the fundamental problem that build system flags are not obeyed.

We can also skip the "ldflags" tests (```INSANE_SKIP_${PN} += "ldflags"```).